snuba/consumers/consumer_builder.py

```diff
@@ -62,6 +65,33 @@ class MockParameters:
     std_deviation: int
 
 
+def get_sliced_snuba_topic(
```

Comment:
Since this and the method below work on sliced and non-sliced topics, should they just be called `get_snuba_topic/get_arroyo_topic`? The majority of calls will be for non-sliced topics

Reply:
It's true the methods work also for non-sliced topics. I think it could be helpful to call them something like `get_physical_snuba_topic`/`get_physical_arroyo_topic` though?

Reply:
sounds good